### PR TITLE
Atalay/assign test fix

### PR DESCRIPTION
**Pull request by Atalay-Polat**

### CODE REVIEW

File: exalearn-api/ExaLearn.WebApi/Controllers/UserController.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ public async Task<IActionResult> GetAllUsers()
             return Ok(await _userService.GetAllAsync());
         }
 
-        [HttpGet("{id}/myAssignedTests")]    // id must be at the end example: getUser{id}
+        [HttpGet("myAssignedTests/{id}")]    // id must be at the end example: getUser{id}
```
Comment (romanstat) [inline exalearn-api/ExaLearn.WebApi/Controllers/UserController.cs]: who said that

File: exalearn-api/ExaLearn.WebApi/Controllers/UserController.cs [hunk 2]
```diff
@@ -52,18 +52,24 @@ public async Task<IActionResult> CreateAssignedTest([FromBody] AssignedTestDTO a
         }
         
 
-        [HttpGet("{id}/allAssignedTest")] // id must be at the end example: getUser{id}
+        [HttpGet("allAssignedTest/{id}")] // id must be at the end example: getUser{id}
         public async Task<IActionResult> GetHrAssignedTestById(int id)
         {
             return Ok(await _userService.GetHrAssignedTestByIdAsync(id)); //remove this method. what is this method for? how does it take all tests if it takes tests by id?
```
Comment (romanstat) [inline exalearn-api/ExaLearn.WebApi/Controllers/UserController.cs]: for whom are the comments?

Comment (Atalay-Polat) [reply exalearn-api/ExaLearn.WebApi/Controllers/UserController.cs]: Vlad

Comment (Atalay-Polat) [reply exalearn-api/ExaLearn.WebApi/Controllers/UserController.cs]: I've returned old endpoints

**Comment by romanstat** [comment]
remove comments

**Review by romanstat — APPROVED** [review]

**Review by AlpaslanPolat — APPROVED** [review]

**Pull request merged**